contracts/package.json

```diff
@@ -33,7 +33,7 @@
     "ethereum-waffle": "^3.2.0",
     "ethers": "^5.0.17",
     "fs-extra": "^9.0.1",
-    "hardhat": "^2.6.0",
+    "hardhat": "^2.6.8",
```

Comment:
If I change network, I see an error iff:

1. I change using Metamask, not our selector
2. I'm on the Send page

![umbra-network-change-send](https://user-images.githubusercontent.com/1479802/145410967-e7dd81f6-bb84-4c78-933f-f84b98d7d394.gif)

Reply:
fixed in 480da6e